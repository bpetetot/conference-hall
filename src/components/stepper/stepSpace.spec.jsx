/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import StepSpace from './stepSpace'

describe('components/stepSpace', () => {
  it('should render when step is current', () => {
    const wrapper = shallow(<StepSpace step={0} current={0} />)
    expect(wrapper).toMatchSnapshot()
  })
  it('should render when current is one before step', () => {
    const wrapper = shallow(<StepSpace step={10} current={9} />)
    expect(wrapper).toMatchSnapshot()
  })
  it('should render when current is lower than step', () => {
    const wrapper = shallow(<StepSpace step={10} current={1} />)
    expect(wrapper).toMatchSnapshot()
  })
})
