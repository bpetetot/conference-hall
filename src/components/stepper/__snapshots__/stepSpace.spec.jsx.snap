// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/stepSpace should render when current is lower than step 1`] = `
<div
  className="step-space"
/>
`;

exports[`components/stepSpace should render when current is one before step 1`] = `
<div
  className="step-space"
/>
`;

exports[`components/stepSpace should render when step is current 1`] = `
<div
  className="step-space space-active-to-default"
/>
`;
